api/src/main/java/datawave/webservice/response/objects/KeyBase.java

```diff
@@ -6,11 +6,8 @@
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlSeeAlso;
 
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
-
 import datawave.webservice.query.result.event.HasMarkings;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY)
```

Comment:
This was removed to try and fix enunciate issues, but I think the query microservice PRs disable enunciate...  Need to look into this.

Reply:
I wouldnt be opposed to removing enunciate in favor of just about anything else

Reply:
yeah, we recently removed it from integration, so im not going to worry about it.